### PR TITLE
Changes for implementing Guestlists tab. #66

### PR DESCRIPTION
**Pull request by mayur43s**
This pull request contains the changes for implementing Guestlist tab #66 and also contains the updated sounds #14.

### CODE REVIEW

File: Bouncer/Controllers/SettingsViewController.swift [hunk 1]
```diff
@@ -119,19 +119,17 @@ class SettingsViewController: BaseViewController, UITableViewDelegate, UITableVi
...
+            cell.labelValue.text = arrTitleValues[indexPath.row]
         }
         else {
-            cell.lblName.text = arrTitleSupport[indexPath.row]
+            cell.labelName.text = arrTitleSupport[indexPath.row]
             
             let version = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String
             let bundleVersion = Bundle.main.infoDictionary?["CFBundleVersion"] as? String
-            cell.lblValue.text = version?.appending(" (\(bundleVersion!))")
+            cell.labelValue.text = version?.appending(" (\(bundleVersion!))")
```
Comment (kidbombay) [inline Bouncer/Controllers/SettingsViewController.swift]: Huh? What's this do?

File: Bouncer/Controllers/TicketListContactDetailsViewController.swift [hunk 2]
```diff
@@ -171,7 +171,7 @@ class TicketListContactDetailsViewController: BaseViewController, UITableViewDel
     func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
         
         let headerView = UIView(frame: CGRect(x: 0, y: 0, width: tableView.frame.size.width, height: 44))
-        headerView.backgroundColor = UIColor.darkColor.withAlphaComponent(0.5)
+        headerView.backgroundColor = UIColor.color(intRed: 26, green: 29, blue: 37, alpha: 1.0)
```
Comment (kidbombay) [inline Bouncer/Controllers/TicketListContactDetailsViewController.swift]: In a future version let's create a colors constants file.

File: Bouncer/Event.storyboard [hunk 3]
```diff
@@ -759,6 +768,8 @@
     <resources>
         <image name="back_Arrow" width="13" height="23"/>
         <image name="cross" width="16" height="16"/>
+        <image name="feed_tab" width="20" height="14"/>
```
Comment (kidbombay) [inline Bouncer/Event.storyboard]: this should be named guestlist tab not feed tab.

File: Bouncer/Controllers/SettingsViewController.swift (hunk 1 above)
Comment (mayur43s) [inline Bouncer/Controllers/SettingsViewController.swift]: I've updated the variable name here.

File: Bouncer/Controllers/TicketListContactDetailsViewController.swift (hunk 2 above)
Comment (mayur43s) [inline Bouncer/Controllers/TicketListContactDetailsViewController.swift]: We already have a color helper method which have some predefined colors. But this one was being used on a single place that's why its not there in that file.

File: Bouncer/Event.storyboard (hunk 3 above)
Comment (mayur43s) [inline Bouncer/Event.storyboard]: I'll change this.

**Pull request merged**